### DISCUSSION TITLE
Disabling 2fa for a user

### DISCUSSION THREAD

**Discussion by tomkolp** [WeblateOrg/weblate#12248] [General] [answered]
I have activated 2FA on my admin account and generated backup keys. What to do if a user loses keys and loses the ability to authorize using 2FA? Can an admin disable 2FA on a user account?

**Comment by nijel** [comment, marked as answer]
It was not yet present, I've just implemented it in edc4965b4c55ed1d83e54b060d9e1dff306ce830
